keyboards/dekunukem/duckypad/duckypad.h

```diff
@@ -24,9 +24,9 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 #define LAYOUT( \
     k00, k01, k02, \
-    k10, k11, k12, \
-    k20, k21, k22, \
-    k30, k31, k32, \
-    k40, k41, k42, \
-    kDown, kUp \
-) {{k00, k01, k02, k10, k11, k12, k20, k21, k22, k30, k31, k32, k40, k41, k42, kDown, kUp}}
+    k03, k04, k05, \
```

Comment:
Why was this changed?

Reply:
My direct pins are not setup in a matrix so I felt this better represented the matrix defined in config.h